### ISSUE TITLE
3.2.5 pro 3.3.1 All Site Setting get deleted after several hours

### ISSUE THREAD

**Issue by drhaldun** [elementor/elementor#15527]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.20.1
	MySQL version: Debian 10 v10.3.29
	PHP Version: 7.4.20
	PHP Max Input Vars: 1000
	PHP Max Post Size: 250M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: https://acilci.net
	Home URL: https://acilci.net
	WP Multisite: No
	Max Upload Size: 200 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: tr
	Timezone: Europe/Istanbul
	Debug Mode: Inactive

== Theme ==
	Name: JupiterX Child
	Version: 1.0.0
	Author: Artbees
	Child Theme: Yes
	Parent Theme Name: JupiterX
	Parent Theme Version: 1.25.1
	Parent Theme Author: Artbees

== User ==
	Role: administrator
	WP Profile lang: tr_TR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15

== Active Plugins ==
	Academic Blogger's Toolkit
		Version: 5.2.2
		Author: Derek P Sifford

	Ad Inserter
		Version: 2.7.1
		Author: Igor Funa

	Admin Columns
		Version: 4.3.2
		Author: AdminColumns.com

	Adminimize
		Version: 1.11.7
		Author: Frank Bültge

	Advanced Custom Fields
		Version: 5.9.7
		Author: Delicious Brains

	Akismet Anti-Spam
		Version: 4.1.9
		Author: Automattic

	Blubrry PowerPress
		Version: 8.6.6
		Author: Blubrry

	Broken Link Checker
		Version: 1.11.15
		Author: WPMU DEV

	Dynamic Content for Elementor
		Version: 1.14.5
		Author: Dynamic.ooo

	eForm - WordPress Form Builder
		Version: 4.15.0
		Author: WPQuark

	Elementor
		Version: 3.2.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.3.1
		Author: Elementor.com

	Genesis Blocks
		Version: 1.2.4
		Author: StudioPress

	GoodBarber Api
		Version: 1.0.19
		Author: GoodBarber

	iThemes Security
		Version: 7.9.1
		Author: iThemes

	JetBlocks For Elementor
		Version: 1.2.8
		Author: Crocoblock

	JetBlog For Elementor
		Version: 2.2.13
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.5.7
		Author: Crocoblock

	Jetpack
		Version: 9.8.1
		Author: Automattic

	JetSearch For Elementor
		Version: 2.1.11
		Author: Crocoblock

	JetSmartFilters
		Version: 2.3.0
		Author: Crocoblock

	Jupiter X Core
		Version: 1.22.2
		Author: Artbees

	Loco Translate
		Version: 2.5.3
		Author: Tim Whitlock

	Newsletter
		Version: 7.2.0
		Author: Stefano Lissa & The Newsletter Team

	Newsletter - Addons Manager
		Version: 1.1.4
		Author: The Newsletter Team

	Newsletter - Amazon SES
		Version: 1.3.2
		Author: The Newsletter Team

	Newsletter - Automated Newsletters
		Version: 4.5.1
		Author: The Newsletter Team

	Newsletter - Autoresponder
		Version: 1.3.1
		Author: The Newsletter Team

	Newsletter - Extended Composer Blocks
		Version: 1.3.3
		Author: The Newsletter Team

	Newsletter - Reports and Retargeting
		Version: 4.4.0
		Author: The Newsletter Team

	Newsletter - WP Users Integration
		Version: 1.2.7
		Author: The Newsletter Team

	Posts Table Pro
		Version: 2.2.1
		Author: Barn2 Plugins

	PublishPress Capabilities Pro
		Version: 2.0.1
		Author: PublishPress

	PublishPress Pro
		Version: 3.3.0
		Author: PublishPress

	Query Monitor
		Version: 3.7.1
		Author: John Blackbourn

	Search & Filter Pro
		Version: 2.5.6
		Author: Code Amp

	StockPack – Stock images in WordPress
		Version: 3.3.0
		Author: Derikon Development

	TablePress
		Version: 1.13
		Author: Tobias Bäthge

	Ultimate Member
		Version: 2.1.21
		Author: Ultimate Member

	Visual Link Preview
		Version: 2.2.1
		Author: Bootstrapped Ventures

	WP-Optimize - Clean, Compress, Cache
		Version: 3.1.11
		Author: David Anderson, Ruhani Rabin, Team Updraft

	WP Amazon SES SMTP
		Version: 1.0.5
		Author: FormGet

	WP Crontrol
		Version: 1.10.0
		Author: John Blackbourn & crontributors

	WP Dashboard Notes
		Version: 1.0.10
		Author: Jeroen Sormani

	WP Fastest Cache
		Version: 0.9.1.9
		Author: Emre Vona

	WP LinkedIn Auto Publish
		Version: 7.4
		Author: Martin Gibson

	Yoast SEO Premium
		Version: 15.2.1
		Author: Team Yoast


== Elementor - Compatibility Tag ==
	
	Dynamic Content for Elementor: Compatibility not specified
	Elementor Pro: Compatibility not specified
	JetBlocks For Elementor: Compatibility not specified
	JetBlog For Elementor: Compatibility not specified
	JetElements For Elementor: Compatibility not specified
	JetSearch For Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	archive : 8
		archive-posts : 8
		button : 6
		divider : 16
		form : 6
		heading : 16
		html : 1
		image : 6
		posts : 6
		raven-posts : 1
		social-icons : 2
		text-editor : 12
		theme-archive-title : 9
	footer : 1
		image : 6
		social-icons : 1
		text-editor : 1
	header : 4
		button : 1
		heading : 1
		icon : 2
		icon-list : 1
		image : 10
		nav-menu : 2
		posts : 2
		raven-nav-menu : 3
		search-form : 3
		social-icons : 2
		text-editor : 2
	page : 11
		author-box : 35
		button : 5
		dce-user-fields : 1
		divider : 23
		dyncontel-date : 1
		dyncontel-featured-image : 1
		dyncontel-terms : 1
		dyncontel-title : 1
		form : 2
		google_maps : 2
		heading : 56
		html : 2
		icon : 3
		image : 44
		jet-blog-video-playlist : 1
		jet-button : 3
		posts : 3
		raven-countdown : 4
		raven-post-comments : 3
		raven-posts-carousel : 2
		shortcode : 4
		text-editor : 23
		video : 7
	popup : 8
		button : 3
		form : 2
		gallery : 2
		heading : 7
		icon-list : 2
		image : 5
		login : 1
		raven-search-form : 1
		social-icons : 2
		spacer : 2
		text-editor : 7
	section : 13
		button : 6
		divider : 34
		form : 6
		heading : 38
		html : 1
		image : 14
		jet-smart-filters-checkboxes : 1
		jet-smart-filters-search : 1
		post-info : 1
		posts : 53
		share-buttons : 1
		social-icons : 2
		spacer : 6
		table-of-contents : 1
		text-editor : 7
	single : 1
		author-box : 2
		breadcrumbs : 1
		dce-add-to-favorites : 1
		heading : 3
		jet-blog-posts-navigation : 2
		post-comments : 1
		raven-button : 1
		raven-image : 1
		shortcode : 1
		text-editor : 3
		theme-post-content : 1
		theme-post-title : 1
	widget : 0
		jet-ajax-search : 1
		post-navigation : 1
	wp-page : 13
		author-box : 20
		button : 7
		dce-dynamicposts-v2 : 1
		dce-views : 1
		divider : 27
		heading : 19
		html : 15
		image : 39
		jet-blog-video-playlist : 6
		jet-button : 2
		jet-smart-filters-checkboxes : 1
		jet-smart-filters-pagination : 2
		jet-smart-filters-search : 1
		jet-smart-filters-select : 1
		posts : 5
		raven-button : 2
		raven-countdown : 2
		raven-heading : 8
		raven-post-comments : 3
		raven-posts : 4
		shortcode : 3
		text-editor : 2
		video : 5
	search-results : 2
		archive-posts : 1
		html : 2
		posts : 1
		raven-search-form : 1
	error-404 : 3
		button : 2
		heading : 2
		text-editor : 2


== Log ==
	
Log: showing 20 of 372021-05-07 13:27:24 [info] Dynamic Content For Elementor/Upgrades - _v_1_14_0_update_options Start  
2021-05-07 13:27:24 [info] Dynamic Content For Elementor/Upgrades - _v_1_14_0_update_options Finished 
2021-05-07 13:27:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic Content For Elementor',
  'from' => '1.13.6',
  'to' => '1.14.4',
)]
2021-05-12 10:44:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Dynamic Content For Elementor',
  'from' => '1.14.4',
  'to' => '1.14.5',
)]
2021-05-27 17:06:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.3',
  'to' => '3.2.4',
)]
2021-06-08 22:16:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.3.0',
)]
2021-06-08 22:16:37 [info] elementor-pro::elementor_pro_updater Started 
2021-06-08 22:16:37 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-06-08 22:16:38 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-06-08 22:16:38 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-06-08 22:16:40 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2021-06-08 22:16:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.3.0',
)]
2021-06-08 22:16:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.2',
  'to' => '3.3.0',
)]
2021-06-18 11:39:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.4',
  'to' => '3.2.5',
)]
2021-06-25 15:17:35 [info] elementor-pro::elementor_pro_updater Started 
2021-06-25 15:17:35 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-06-25 15:17:35 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-06-25 15:17:35 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-06-25 15:17:37 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2021-06-25 15:17:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.0',
  'to' => '3.3.1',
)]

PHP: showing 4 of 4PHP: 2021-05-07 13:54:37 [error X 1][/var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::281] Uncaught Error: Call to a member function update_meta() on bool in /var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:281
Stack trace:
#0 /var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(162): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#3 /var/www/vhosts/acilci.net/httpdocs/wp-includes/class-wp-hook.php(292): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /var/www/vhosts/acilci.net/httpdocs/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters()
#5 /var/www/vhosts/acilci.net/httpdocs/wp-includes/plugin.php(484): WP_Hook->do_action()
#6 /var/www/vhosts/acilci.n [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-05-07 23:23:40 [notice X 63][/var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/post-featured-image.php::36] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-06-17 13:08:11 [warning X 4][/var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php::694] array_merge(): Expected parameter 2 to be an array, int given [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-06-18 12:15:57 [notice X 1][/var/www/vhosts/acilci.net/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2021-06-18 08:41:29 [error X 4][https://acilci.net/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.2.5:2:33398] An invalid argument supplied as multiple control value [array (
  'category' => 'InvalidElementData',
  'tag' => 'Element `undefined` got as `custom_thumb` value. Expected array or object.',
)]


				
```
</details>

**Comment by shilo-ey** [comment]
A fix is already present in Elementor v3.3.0 which is in beta currently. 
It will prevent the default kit from being recreated and replaced.

Thanks

**Comment by drhaldun** [comment]
How and where can i find the default kit template and recover it?


Prof. Dr. Haldun AKOĞLU
Marmara Üniversitesi Tıp Fakültesi, Acil Tıp Anabilim ***@***.***

Yönetim Kurulu Üyesi, Türkiye Acil Tıp Derneği, @trTATD
***@***.***_net
Editör, Turkish Journal of Emergency Medicine @TurkJEmergMed

Twitter: @IstanbulEMDoc
***@***.******@***.***
